apps/wasm/mina-berkeley/src/js_exports.rs

```diff
@@ -0,0 +1,46 @@
+// Copyright 2020 ChainSafe Systems
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::{logger::JsExportableLogger, *};
+use wasm_bindgen::prelude::*;
+
+#[wasm_bindgen]
+pub async fn run_processor() {
+    frontier::PROCESSOR_BERKELEY.run().await;
+}
+
+#[wasm_bindgen]
+pub async fn set_api_list(api_urls: String) -> Result<(), JsError> {
+    let mut backend = frontier::PROCESSOR_BERKELEY.nonconsensus_ops_mut().await;
+    let api_list: Vec<String> = api_urls.split('\n').map(|s| s.to_string()).collect();
+    backend.set_api_list(api_list.into_iter());
+    Ok(())
+}
+
+#[wasm_bindgen]
+pub async fn set_tracking_accounts(accounts: String) -> Result<(), JsError> {
+    let mut backend = frontier::PROCESSOR_BERKELEY.nonconsensus_ops_mut().await;
+    let account_list: Vec<String> = accounts.split('\n').map(|s| s.to_string()).collect();
+    backend.set_tracking_accounts(account_list.into_iter());
+    Ok(())
+}
+
+#[wasm_bindgen]
+pub async fn poll_latest_blocks_once() -> Result<(), JsError> {
```

Comment:
There should be an api to run the process to kick off the event loop

Reply:
this one?
https://github.com/ChainSafe/mina-rs/blob/7bbf4bd36e7dd4ec886c40666a60ed8ae8d42abb/apps/wasm/mina-berkeley/src/js_exports.rs#L8

Reply:
Sry did not see that